packages/atomic/src/components/search/search-box-suggestions/atomic-search-box-instant-results/atomic-search-box-instant-results.tsx

```diff
@@ -116,7 +118,10 @@ export class AtomicSearchBoxInstantResults implements InitializableComponent {
   }
 
   private renderItems(): SearchBoxSuggestionElement[] {
-    if (!this.bindings.suggestedQuery() || this.bindings.store.isMobile()) {
+    if (
+      !this.bindings.suggestedQuery() ||
+      (this.bindings.store as AtomicStore).isMobile()
```

Comment:
🤔 why the cast?

Reply:
Good catch; 

Just an artefact from the refactor that is no longer necessary with the changes to `SearchBoxSuggestionsBindings` interface to make it generic. 

Forgot to cleanup